src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag

```diff
@@ -88,13 +88,17 @@
             <label for="studentNationality">
                 Nationality
             </label>
-            <input id="studentNationality"
-                   name="<%= Const.ParamsNames.STUDENT_NATIONALITY %>"
-                   class="form-control"
-                   type="text"
-                   data-actual-value="${profile.nationality}"
-                   value="${profile.nationality}"
-                   placeholder="Nationality">
+            <select id="studentNationality"
+                    name="<%=Const.ParamsNames.STUDENT_NATIONALITY%>"
+                    class="form-control" style="width: 300px">
+                <option label="--- Select ---" />
```

Comment:
`/>` is not compliant to our style guide. Close it with proper closing tag, i.e `</option>`.